rvr/lib/sphero_rvr.py

```diff
@@ -0,0 +1,170 @@
+import board
+import busio
+import time
+import struct
+
+class LEDs:
+    RIGHT_HEADLIGHT = [0x00, 0x00, 0x00, 0x07]
+    LEFT_HEADLIGHT = [0x00, 0x00, 0x00, 0x38]
+    LEFT_STATUS = [0x00, 0x00, 0x01, 0xC0]
+    RIGHT_STATUS = [0x00, 0x00, 0x0E, 0x00]
+    BATTERY_DOOR_FRONT = [0x00, 0x03, 0x80, 0x00]
+    BATTERY_DOOR_REAR = [0x00, 0x00, 0x70, 0x00]
+    POWER_BUTTON_FRONT = [0x00, 0x1C, 0x00, 0x00]
+    POWER_BUTTON_REAR = [0x00, 0xE0, 0x00, 0x00]
+    LEFT_BRAKELIGHT = [0x07, 0x00, 0x00, 0x00]
+    RIGHT_BRAKELIGHT = [0x38, 0x00, 0x00, 0x00]
```

Comment:
Keep these for later

Reply:
This looks like copied from the sphero.py for the micro:bit implementation of micropython from 2019